cli/studio.py

```diff
@@ -71,7 +71,7 @@ def _build_single_scenario(clean: bool, allow_offset_map: bool, scenario: str):
     from smarts.core.scenario import Scenario
 
     traffic_histories = Scenario.discover_traffic_histories(scenario_root_str)
-    shift_to_origin = not allow_offset_map or bool(traffic_histories)
+    shift_to_origin = not allow_offset_map
```

Comment:
I would go a step further here.  I suggest forcing it to not shift the map when there's traffic history data:
```python
    # don't shift maps for scenarios with traffic histories since history data must line up with map
    shift_to_origin = not allow_offset_map and not bool(traffic_histories)
```

Then, where the `--allow-offset-map` option is documented just above that, you should amend it to be something like:
```python
@click.option(
    "--allow-offset-maps",
    is_flag=True,
    default=False,
    help="Allows road networks (maps) to be offset from the origin. If not specified, a new network file is created if necessary.  Defaults to False except when there's Traffic History data associated with the scenario.",
)
```